proto/protovalidate/buf/validate/validate.proto

```diff
@@ -266,13 +261,13 @@ message PredefinedConstraints {
   // }
   // ```
   repeated Constraint cel = 1;
+
+  reserved 24, 26;
```

Comment:
Think we should reserve the names? It's unlikely we'd reintroduce them, but it would be precautionary.

Reply:
Done.